src/theme/variables/edgeDark.js

```diff
@@ -188,6 +188,17 @@ export const edgeDark: Theme = {
 
   // Misc
   // pressedOpacity: 0.25, // Should be removed when press colors are given to buttons and links
+  modalIcon: palette.edgeBlue,
+
+  // Scene/Components background
+  promoCardBackgroud: palette.edgeBlue,
```

Comment:
promoCardBackgroud Should be promoCardBackground

Reply:
removed